home/tests/test_models.py

```diff
@@ -983,3 +983,9 @@ def test_clean_identical_min_max(self):
             "min cannot be greater than max",
             e.exception.message,
         )
+        with self.assertRaises(ValidationError) as e:
+            IntegerQuestionBlock().clean(self.create_min_max_value(min=-50, max=40))
```

Comment:
I would add another separate assert where max is negative too.

Right now the code on the other side of the OR is untested.